lists/1313161554/list.json

```
@@ -777,6 +777,14 @@
       "decimals": 18,
       "chainId": 1313161554,
       "logoURI": "https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/0xa69d9Ba086D41425f35988613c156Db9a88a1A96/logo.svg"
-    }
+    },
+    {
+      "name": "BNB - Allbridge",
+      "address": "0xb14674c7264ec7d948b904aab2c0e0f906f6e762",
```

Please use checksummed address
```suggestion
      "address": "0xb14674C7264eC7d948B904Aab2c0E0F906F6e762",
```